media/js/firefox/family-nav.js

```diff
@@ -213,6 +220,55 @@ if (typeof window.Mozilla === 'undefined') {
             $ctaWrapper.append($('#' + ctaId));
         };
 
+        // analytics
+        var _initGA = function() {
+            // clicks on top level nav links
+            $primaryLinks.on('click', function(e) {
+                var $this = $(this);
+
+                if (e.metaKey || e.ctrlKey) {
+                    window.gaTrack(['_trackEvent', 'Fx Family Nav Interactions', $this.data('id'), $this.data('id') + ' - top nav link']);
+                } else {
+                    e.preventDefault();
+                    window.gaTrack(['_trackEvent', 'Fx Family Nav Interactions', $this.data('id'), $this.data('id') + ' - top nav link'], function() {
+                        window.location = $this.attr('href');
+                    });
+                }
+            });
+
+            // clicks on subnav links
+            $subNavs.on('click', 'a', function(e) {
+                var $this = $(this);
+
+                var parentName = $this.data('id').split('-');
```

Comment:
This feels a little brittle as the code depends on a `-` in the markup? How about we also check the length of `parentName` before sending the GA event.